app/lib/esClient.js

```diff
@@ -61,4 +61,5 @@ module.exports = {
   getPharmacies,
   getOpenPharmacies,
   getHealth,
+  client,
```

Comment:
I'm not keen on changes to the API to include things we don't want using in production. I think the tests should `require('elasticsearch')` There would be some code duplication but, for me this is the lesser of the two evils.

Reply:
> I'm not keen on changes to the API

Which API does this refer to?

> things we don't want using in production

`client` is used in production.

Given the configuration of the client is the same for both core code and test code, and exposing the client in a configured instance (ideally in its own module*) is a logical refactoring to provide a single responsibility from that module I'd be inclined to use that in all scenarios. And not have a test specific instance.

\* Refactor to this

Reply:
By API I mean the API of `esClient`.

AFAIK the only place `esClient.client` is used is in the tests.

I'd suggest that the `before` should be:

```
const elasticSearch = require('elasticsearch');
const esConfig = require('../../config/config').es;
const client = elasticsearch.Client({ host: `${esConfig.host}:${esConfig.port}` });

before('set up data', async () => {
        await client.update({
        <snip>
        });
}
```

The reason for this is that the exposing of `client` is for testing only. I suspect we would be unhappy if client was used directly in production code rather than using an intent specific method like `getOpenPharmacies`.